.github/workflows/build-test-docker.yaml

```diff
@@ -2,26 +2,85 @@ name: build-test-docker
 
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
Please add a toplevel comment explaining the goal of this workflow (you can see at the other workflow in the same directory for example).